plateau/core/_time.py

```diff
@@ -21,4 +22,6 @@ def datetime_utcnow():
 
     Same as datetime.datetime.utcnow
     """
-    return datetime.datetime.utcnow()
+    if sys.version_info <= (3, 10):
```

Comment:
```suggestion
    if sys.version_info < (3, 11):
```
Otherwise this evaluates to false for all patch releases of 3.10.